package.json

```diff
@@ -3,17 +3,19 @@
   "version": "2.0.0",
   "private": true,
   "description": "",
-  "main": "index.js",
   "scripts": {
     "test": "jest",
     "test:ci": "yarn test -- --ci",
     "test:watch": "yarn test -- --watch",
     "coverage": "yarn test -- --coverage",
     "coverage:ci": "yarn test:ci -- --coverage",
-    "tslint": "tslint -c tslint.json 'src/**/*.ts{,x}'",
-    "tslint:fix": "tslint -c tslint.json 'src/**/*.ts{,x}' --fix",
+    "tslint": "tslint -c tslint.json 'src/**/*.ts{,x}' --exclude 'src/shared/proto/**'",
```

Comment:
Without this tslint takes _much_ longer trying to lint the generated `messages.js` (which doesn't conform to our style guide).